Duplicati/CommandLine/Program.cs

```diff
@@ -163,31 +163,16 @@ public static int RealMain(string[] args)
                 }
 
             // Probe for "help" to avoid extra processing
-            bool isHelp = cargs.Count == 0 || (cargs.Count >= 1 && string.Equals(cargs[0], "help", StringComparison.OrdinalIgnoreCase));
-
-            if (isHelp) 
+            if (cargs.Count == 0 || (string.Equals(cargs[0], "help", StringComparison.OrdinalIgnoreCase))) 
             {
                 return Commands.Help(outwriter, setup, cargs, options, filter);
             }
 
-            if ((options.ContainsKey("parameters-file") && !string.IsNullOrEmpty("parameters-file")) || (options.ContainsKey("parameter-file") && !string.IsNullOrEmpty("parameter-file")) || (options.ContainsKey("parameterfile") && !string.IsNullOrEmpty("parameterfile")))
+            if (options.ContainsKey("parameters-file") && !string.IsNullOrEmpty(options["parameters-file"]))

```

Comment:
This changes the logic a bit. Previously, it was possible to use `parameter-file`, `parameterfile` and `parameters-file`. With this change, only the last version is accepted.

This could potentially break existing backups, if one of the other names are used. What is the motivation to remove the alias names?

Reply:
My bad, I didn't recognize these were aliases at all :) I reverted this to be backwards compatible, even though it strikes me to have these aliases for an option (other options don't have this). In the new commit I rewrote the code to avoid duplication. Notice that in the old code, it should have checked on the value option[parameterOption] to be not null or empty, instead it check a constant string to be non-empty which would hence always be false.

Reply:
There are some options that have aliases, but it is not common. Maybe it would be better if I had not introduced aliases here, but I figured that users would have a hard time remembering what the option was called, and now it is in there :).